src/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+  
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+  
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+  
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Try to rewrite it by loop for more practice.  If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

Reply:
I returned to the task after a long time with the knowledge of scss technology.
Here is the solution with a "for" loop.
Since the task is not yet able to compile scss files, I will leave a comment with the code:
`@for $index-of-star from 1 through 5 {
    .stars--#{$index-of-star} :nth-child(-n + #{$index-of-star})  {
      background-image: url(./src/images/star-active.svg);
      }
 } `